director/resources/multi_export_topology.json

```diff
@@ -4,8 +4,8 @@
      {
         "caches":[
          {
-            "auth_endpoint": "osdf-uw-cache.svc.osg-htc.org:8443",
-            "endpoint": "osdf-uw-cache.svc.osg-htc.org:8443",
+            "auth_endpoint": "https://osdf-uw-cache.svc.osg-htc.org:8443",
+            "endpoint": "https://osdf-uw-cache.svc.osg-htc.org:8443",
```

Comment:
Good catch -- these were missing schemes before. I think we still want to have an http/https split though.